### PR TITLE
[PBW-4025] Close caption is showing on End screen

### PR DESCRIPTION
**Pull request by brianfwong**
Publishing event to end CC at end of play. The check for ‘Safari’ was
preventing CC on replay as a result of the ending event change.

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -326,10 +326,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.pluginsClickElement.removeClass("showing");
         this.state.screenToShow = CONSTANTS.SCREEN.PLAYING_SCREEN;
         this.state.playerState = CONSTANTS.STATE.PLAYING;
-        if (Utils.isSafari()){
-          //Safari only can set cc when the video is playing, not before
```
Comment (mistermendez) [inline js/controller.js]: do we not need this anymore?

File: js/controller.js [hunk 2]
```diff
@@ -411,6 +408,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onVcPlayed: function(event, source) {
+      var language = "";
```
Comment (mistermendez) [inline js/controller.js]: var language unused?

File: js/controller.js (hunk 1 above)
Comment (brianfwong) [inline js/controller.js]: The issue was that this check was causing "this.setClosedCaptionsLanguage();" to only be done for Safari when it should be done for all browsers.

**Comment by mistermendez** [comment]
@brianfwong conflicts

File: js/controller.js (hunk 2 above)
Comment (brianfwong) [inline js/controller.js]: As part of our intention here to wipe out the data, we set the language to a blank entry so it doesn't display on the endscreen.

**Pull request merged**